cli/src/command/account.rs

```diff
@@ -31,11 +33,38 @@ impl CommandT for Command {
         match self {
             Command::Address(cmd) => cmd.run(ctx).await,
             Command::Balance(cmd) => cmd.run(ctx).await,
+            Command::Create(cmd) => cmd.run(ctx).await,
             Command::Transfer(cmd) => cmd.run(ctx).await,
+            Command::List(cmd) => cmd.run(ctx).await,
         }
     }
 }
 
+#[derive(StructOpt, Debug, Clone)]
+/// Create an account identified by `name`.
+///
+/// Generate a random key pair that will be identified
+/// by the unique `name`.
+///
+/// Fail if there is already an account with the given `name`
+/// locally.
+pub struct Create {
+    name: String,
+}
+
+#[async_trait::async_trait]
+impl CommandT for Create {
+    async fn run(&self, _ctx: &CommandContext) -> Result<(), CommandError> {
+        let (kp, _) = ed25519::Pair::generate();
```

Comment:
should do this instead:

```suggestion
        let (, seed) = ed25519::Pair::generate();
```